rekallpy/rekall/stdlib/ingest.py

```diff
@@ -0,0 +1,331 @@
+    example, if the mapping key, t1, t2, x1, and x2 values are stored in fields
+    ``video_id``, ``min_frame``, ``max_frame``, ``bbox_x1``, and ``bbox_x2``
+    fields respectively, ``bounds_schema`` should be set to::
+
+        {
+            'key': 'video_id',
+            't1': 'min_frame',
+            't2': 'max_frame',
+            'x1': 'bbox_x1',
+            'x2': 'bbox_x2'
+        }
+
+    And for each item in iterable, the key, t1, t2, x1, and x2 values will be
+    accessed by ``key_accessor(item, 'video_id')``,
+    ``key_accessor(item, 'min_frame')``, ``key_accessor(item, 'max_frame')``,
+    ``key_accessor(item, 'bbox_x1')``, ``key_accessor(item, 'bbox_x2')``,
+    respectively.
+
+    If only the key needs to be updated, ``bounds_schema`` should be set to::
+
+        {
+            'key': 'video_id'
+        }
+
+    In this case, Bounds3D will use default values for x1, x2, y1, and y2.
+
+    Args:
+        iterable: An iterable of elements whose relevant fields can be
+            accessed by ``key_accessor``.
+        key_accessor: A function that takes an element of ``iterable`` and
+            a key and returns the value of the key on that element.
+        bounds_schema (optional): A dictionary that overrides default field
+            keys (``'K'`` for the key, ``'t1'`` for co-ordinate t1, ``'t2'``
+            for co-ordinate t2, ``'x1'`` for co-ordinate x1, ``'x2'`` for
+            co-ordinate x2, ``'y1'`` for co-ordinate y1, and ``'y2'`` for
+            co-ordinate y2).
+        with_payload (optional): A function that takes in an item in iterable
+            and returns the payload for the Interval from the item. Defaults to
+            a function that returns ``None``.
+        progress (optional): Whether to display a loading bar from ``tqdm``.
+            Defaults to ``False``.
+        total (optional): Used in conjunction with ``progress`` to optionally
+            display the total number of items in the loading bar if
+            ``progress`` is ``True``.
+
+    Returns:
+        An IntervalSetMapping with Intervals from each item of iterable.
+    """
+    schema_final = {
+        "key": "key",
+        "t1": "t1",
+        "t2": "t2"
+    }
+    schema_final.update(schema)
+    def key_parser(item):
+        return key_accessor(item, schema_final['key'])
+    def bounds_parser(item):
+        args = [
+            key_accessor(item, schema_final['t1']),
+            key_accessor(item, schema_final['t2'])
+        ]
+        kwargs = {}
+        for k in ['x1', 'x2', 'y1', 'y2']:
+            if k in schema_final:
+                kwargs[k] = key_accessor(item, schema_final[k])
+        return Bounds3D(*args, **kwargs)
+    return IntervalSetMapping.from_iterable(iterable, key_parser,
+        bounds_parser, payload_parser, progress, total)
+
+# from Django QS
+def ism_from_django_qs(qs, bounds_class=Bounds3D, bounds_schema={},
```

Comment:
I found it helpful to support nested field names (rekall 0.1 supports it). 

Example: `attrgetter_accessor(row, 'face.frame.number')`

Reply:
Support added in this function